### PR TITLE
add tests and functionality to /api/topics endpoints and handle error…

### PR DESCRIPTION
**Pull request by veramferreira**
…s - ticket 3

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  return db.end();
+});
+
+describe("/api", () => {
+  test("GET - status 200 - responds with a status message", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toEqual({ msg: "all good here!" });
```
Comment (xViolaine) [inline __tests__/app.test.js:26]: We kind of want to check whether /api sends back all the content it should be sending back - e.g. information about the different endpoints

..but also, this is for task 3.5, we are currently on 3, right?

**Review by xViolaine — CHANGES_REQUESTED** [review]
Hey there!
I am not quite sure what the /api is supposed to be doing here, we want to target this endpoint in task 3.5 and then it's also not just supposed to send a 200 back, but instead a list of endpoints - please amend this before resubmitting :)

File: __tests__/app.test.js (hunk 1 above)
Comment (veramferreira) [reply __tests__/app.test.js:26]: Yep, just saw the next ticket.. I was getting ahead of myself (just did what we we've done on the previous sprints and didn't realised we had to so something different for this one). Sorry!

**Review by robert112233 — APPROVED** [review]
Changes look good, merge !

**Pull request merged**